pyproject.toml

```diff
@@ -288,3 +288,10 @@ ignore_messages = [
     "Duplicate implicit target name: \".*\".",
     "Duplicate explicit target name: \".*\".",
 ]
+
+[tool.pytest.ini_options]
+minversion = "7.0"
+junit_family = "xunit2"
```

Comment:
This is the default nowadays and can thus be omitted:

https://docs.pytest.org/en/7.1.x/reference/reference.html#confval-junit_family

Reply:
Not exactly.

> By default, files matching test_*.py and *_test.py will be considered test modules.

We don't have and *_test.py files under `lib` (and only one in the whole project: `galleries/examples/units/evans_test.py`). So, effectively, you'll detect the same files and can leave it out.

Reply:
I didn't go back in the history, but I'm pretty sure we added `python_files` first to avoid running `evans_test.py`, but later we added `testpaths = lib`, which excluded it in a different manner.

Reply:
I'm +0.05 on leaving `python_files =test_*py` to be very explicit what we consider tests. (But OTOH, I likely would not have intoduced it if it was not there).

Reply:
To be clear, I don't think it needs to stay either.